packages/@react-aria/select/src/useSelect.ts

```diff
@@ -121,7 +121,7 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
   let domProps = filterDOMProps(props, {labelable: true});
   let triggerProps = mergeProps(typeSelectProps, menuTriggerProps, fieldProps);
 
-  let valueId = useId();
+  let valueId = `${triggerProps.id}-value`;
```

Comment:
lets add a comment: used to make predictable id's based on the trigger which is already generated, this aids us in testing

Reply:
Great idea, pushed a fix